installer/dbscript/dbscript-2.sql

```diff
@@ -4315,7 +4320,9 @@ VALUES (1, 0, 1, 0, 0, @apiv2_time_time_sheet_config_data_group_id, @admin_role_
        (1, 0, 0, 0, 0, @apiv2_time_config_time_format_data_group_id, @admin_role_id),
        (1, 0, 0, 0, 0, @apiv2_time_config_time_format_data_group_id, @ess_role_id),
        (1, 0, 0, 0, 0, @apiv2_time_validation_project_activity_name_data_group_id, @admin_role_id),
-       (1, 0, 0, 0, 0, @apiv2_time_validation_project_activity_name_data_group_id, @project_admin_role_id);
+       (1, 0, 0, 0, 0, @apiv2_time_validation_project_activity_name_data_group_id, @project_admin_role_id),
+       (1, 0, 0, 0, 0, @apiv2_time_employee_timesheets_data_group_id, @admin_role_id),
+       (1, 0, 0, 0, 0, @apiv2_time_employee_timesheets_data_group_id, @supervisor_role_id);
```

Comment:
```suggestion
       (1, 1, 1, 0, 0, @apiv2_time_employee_timesheets_data_group_id, @admin_role_id),
       (1, 1, 1, 0, 0, @apiv2_time_employee_timesheets_data_group_id, @supervisor_role_id);
```
@VageeshOrangeHRM check this

Reply:
done